flang/runtime/file.h

```diff
@@ -15,6 +15,12 @@
 #include "memory.h"
 #include <cinttypes>
 #include <optional>
+#ifdef _WIN32
```

Comment:
Are these being moved into the header to make `isatty()` visible?

Reply:
Yes